content/v2.0/backup-restore/restore.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Restore data
+seotitle: Restore data in InfluxDB
+description: >
+  Use the `influxd restore` command to restore data and metadata from an InfluxDB
+  backup file set.
+menu:
+  v2_0:
+    parent: Back up & restore data
+weight: 101
+v2.0/tags: [restore]
+related:
+  - /v2.0/backup-restore/backup/
+  - /v2.0/reference/cli/influxd/restore/
+draft: true
+---
+
+Use the `influxd restore` command to restore data and metadata from an InfluxDB backup file set.
+InfluxDB only supports "offline" data restoration, meaning InfluxDB must be stopped
+to safely and successfully restore data.
+
+{{% cloud-msg %}}
+The `influxd restore` command only restores data to InfluxDB OSS, **not {{< cloud-name "short" >}}**.
+{{% /cloud-msg %}}
+
+When restoring data from a backup file set, InfluxDB temporarily moves existing
+data and metadata while the restore process runs.
+After the process completes, the temporary data is deleted.
```

Comment:
think we can drop this bit? ....or expand why you'd want to know this/how it's helpful